src/Devices/Serial_TC.h

```diff
@@ -0,0 +1,85 @@
+// Year-Month-Day Format for DateTime
+// https://en.wikipedia.org/wiki/ISO_8601
+
+#include <Arduino.h>
+#include <RTClib.h>
+
+class TC_Serial {
+private:
+  const float RREF = 430.0;
+  // RTC_PCF8523 rtc;
+
+public:
+  TC_Serial() {
+    Serial.begin(9600);
+  }
+
+  void print_PID(double Kp, double Ki, double Kd, double output) {
+    Serial.print(F("Kp:"));
+    Serial.print(Kp);
+    Serial.print(F(" Ki:"));
+    Serial.print(Ki);
+    Serial.print(F(" Kd:"));
+    Serial.println(Kd);
+    Serial.print(F("PID output (s): "));
+    Serial.println(output / 1000, 1);
+  }
+
+  void print_DateTime(DateTime now) {
```

Comment:
Rename `now` to `dateTime` since it could (possibly) be something other than now.